src/main/java/club/issizler/okyanus/api/Server.java

```diff
@@ -58,30 +57,6 @@
      */
     boolean isMainThread();
 
-    /**
```

Comment:
You're removing these, but what about the mods that use them? Shouldn't you just deprecate them instead?